Deliverables meeting template

choldgraf opened the pull request:
This is a draft issue template that we can use to plan and structure our deliverables sync meetings.

A few highlights:

- Proposes a structure for the deliverables meeting with checklists for things to do.
- Proposes that we use **every other Monday** for this meeting at 8:30AM California time (which comes directly after our monthly team meeting...this is intentional, but let me know if this is too much).
- Reduces our monthly team meeting from 2 hours to 1.5 hours, with the idea that we'll use that last half hour for the deliverables review meeting.

closes #148

sgibson91 commented:
LGTM - just wanted to check on timings as this looks like it'd last an hour, but I thought we'd discussed 30mins?

choldgraf commented:
Good point. Do folks think we can cover that in 30min? If so we could start with that and see how it goes

sgibson91 commented:
I think part of the reason to keep it short is so that we don't get too stuck into technical discussions. The point of these meetings is to sync up with what's done and prioritise what's next, not fix things there and then. But I think we can flex timings to find the rhythm of the team as we figure it out.

choldgraf commented:
In that case I'll stick with 30 minutes since it'll be easier to grow the meeting as-needed rather than shrink it as-needed :-)

choldgraf commented:
OK I've addressed comments, reduced this meeting to 30 minutes and added it to the end of our monthly team meetings. Updated the top comment with this new information. What do folks think?

choldgraf commented:
How does the @2i2c-org/tech-team feel about having our first deliverables meeting on this Monday? Using the structure described here? We can prototype the meeting see how it goes or what we'd like to change?

sgibson91 commented:
I am definitely pro avoiding giant meetings where possible!

damianavila commented on .github/ISSUE_TEMPLATE/meeting-deliverables.md:
```diff
@@ -0,0 +1,42 @@
+### Activity Backlog
+_(5m) Quick review of [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to let team members signal-boost if needed._
+
+- [ ] Quick team report on what they're up to right now, where they could use help.
```
I think there is an overlapping between this point and the team sync issue... Maybe the meeting should happen every other **Tuesday** so everyone has time to look at other's reports and look into the board by themselves?

damianavila commented:
> How does the @2i2c-org/tech-team feel about having our first deliverables meeting on this Monday? Using the structure described here? We can prototype the meeting see how it goes or what we'd like to change?

See my above comment about Tuesdays instead...

> Something else we could discuss is whether we'd like this meeting to be off-cycle with our monthly meetings so that we can meet a bit earlier, and thus be a bit more timezone friendly for all (this would also help us avoid a giant meeting on days with team meetings)

I do think having this off-cycled from the monthly meeting is a good idea...

choldgraf commented:
The latest commit does a couple things:

- Changes the meeting time to Tuesday, off-cycle with our monthly team meetings.
- Updates the first item to reference the team sync since team members should have already filled this out
- I've also added a calendar entry to our team calendar for **next Tuesday at 8am Pacific** (here's [**a timezone converter**](https://arewemeetingyet.com/Los%20Angeles/2000-01-01/08:00/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=))

I think this is ready to go as long as others are OK with the timing etc. Once this is merged I'll create an issue for our team's first deliverables meeting!

sgibson91 reviewed (approved):
🚀

damianavila commented on practices/meetings.md:
```diff
@@ -22,6 +26,14 @@ We use [this HackMD for our team meeting notes and agenda](https://hackmd.io/Y5S
 
+## Deliverables review meetings
+
+The 2i2c team meets **every other Monday** for 30 minutes.
```
```suggestion
The 2i2c team meets **every other Tuesday** for 30 minutes.
```

damianavila reviewed (approved):
LGTM now!

choldgraf commented:
ok! let's close this and I look forward to chatting with you all about deliverables tomorrow :-) (except for @damianavila who is hopefully hanging out in the middle of nowhere ;-) )
